mycroft/client/speech/hotword_factory.py

```diff
@@ -12,26 +12,31 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-import tempfile
 import time
-
-import sys
-from tempfile import NamedTemporaryFile
+from time import sleep
 
 import os
-from os.path import dirname, exists, join, abspath, expanduser, isdir, isfile
-
-from os import mkdir, getcwd, chdir
-from time import time as get_time
-
-from mycroft.configuration import Configuration
-from subprocess import Popen, PIPE, call
-from threading import Thread
-
+import platform
+import posixpath
+import tempfile
+import requests
+from contextlib import suppress
+from glob import glob
+from os.path import dirname, exists, join, abspath, expanduser, isfile, isdir
+from petact import install_package
+from shutil import rmtree
+from threading import Timer, Event, Thread
+from urllib.error import HTTPError
+
+from mycroft.configuration import Configuration, LocalConf, USER_CONFIG
 from mycroft.util.log import LOG
 
-
 RECOGNIZER_DIR = join(abspath(dirname(__file__)), "recognizer")
+INIT_TIMEOUT = 10
```

Comment:
I try to identify the units of number as often as possible, at least in the initialization.  This prevents things from [crashing spectacularly due to misunderstandings](https://www.wired.com/2010/11/1110mars-climate-observer-report/).

So either name it INIT_TIMEOUT_SECS or just append "  # seconds" on this line.